eng/scripts/Create-Apiview-Token-Python.ps1

```diff
@@ -0,0 +1,10 @@
+[CmdletBinding()]
```

Comment:
Looks like you push this change via the wrong branch which contains other changes as well.

Reply:
I just realized it after I pushed one more change. Sorry about that.